tasks/modules.py

```diff
@@ -180,6 +180,8 @@ def dependency_path(self, agent_version):
     "comp/otelcol/otlp/testutil": GoModule("comp/otelcol/otlp/testutil", independent=True, used_by_otel=True),
     "comp/otelcol/converter/def": GoModule("comp/otelcol/converter/def", independent=True, used_by_otel=True),
     "comp/otelcol/converter/impl": GoModule("comp/otelcol/converter/impl", independent=True, used_by_otel=True),
+    "comp/otelcol/configstore/def": GoModule("comp/otelcol/configstore/def", independent=True, used_by_otel=True),
+    "comp/otelcol/configstore/impl": GoModule("comp/otelcol/configstore/impl", independent=True, used_by_otel=True),
```

Comment:
nit so feel free to ignore - it's not sorted alphabetically

Reply:
Addressed in separate PR: https://github.com/DataDog/datadog-agent/pull/27337 😃